app/components/ruleRunner.js

```diff
@@ -1,8 +1,12 @@
-export const ruleRunner = (fieldName, fieldLabel, ...validation) => {
-  return;
-}
-
+import * as Rules from './errorMessages';
 
+export const ruleRunner = (fieldName, fieldLabel, ...validation) => {
+  // ...validation is an arr of funcs.
+  // loop through and call each validator to get the error messages.
+  for (let i = 0; i < validation.length; i++) {
+    console.log(validation[i]);
+  }
```

Comment:
there is an implementation of this function that you can reference in the article you linked.